lerna.json

```diff
@@ -4,7 +4,8 @@
     "lib/octicons_node",
     "lib/octicons_gem",
     "lib/octicons_helper",
-    "lib/jekyll-octicons"
+    "lib/octicons_jekyll",
+    "lib/octicons_react"
```

Comment:
About this list, I did this because they needed to be built in a specific order, `octicons_node` and `octicons_gem` before the other ruby packages. 

I could see moving `octicons_react` up to below `octicons_node` then if rubygems is down, it won't effect our node packages being published.